config.json

```diff
@@ -251,6 +251,12 @@
       ]
     },
     {
+      "slug": "zebra-puzzle",
+      "difficulty": 1,
+      "topics": [
+      ]
+    },
+    {
```

Comment:
We probably need to do a major reordering of exercises, so that it is easier to figure out where to put a new one...

Considering that this exercise has an example solution with 113 lines, maybe it is a little hard to be be in this position. But I'm not sure because I didn't solve it.